fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSlotReference.java

```diff
@@ -586,4 +612,59 @@ public boolean hasGroupBy() {
             return false;
         }
     }
+
+    private List<NamedExpression> adjustNullableForProjects(
```

Comment:
leave behind some comment for adjestNullableXxx

Reply:
done